projects/demo/src/modules/experimental/button-group-card/examples/1/index.html

```diff
@@ -0,0 +1,70 @@
+<div
+    tuiButtonGroupCard
+    tuiSurface="elevated"
+>
+    <button
+        appearance="flat"
```

Comment:
I have recently added `appearance="link"` for blue buttons with no hover/active effect. I think we should use it instead.

Reply:
but should I override style for dark?

<img width="391" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/85780bb8-9869-4e7c-97d0-e4eed48f5d50">